board/safety/safety_honda.h

```diff
@@ -196,8 +198,13 @@ static int honda_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
 
   // disallow actuator commands if gas or brake (with vehicle moving) are pressed
   // and the the latching controls_allowed flag is True
-  int pedal_pressed = gas_pressed_prev || (gas_interceptor_prev > HONDA_GAS_INTERCEPTOR_THRESHOLD) ||
+  int pendal_pressed = 0;
+  if (unsafe_mode & UNSAFE_DISABLE_DISENGAGE_ON_GAS) {
```

Comment:
Use unsafe_allow_gas

Reply:
pendal_pressed is also misspelt. And can you rebase on master, sorry I introduced a misra violation.